Formula/nlopt.rb

```diff
@@ -13,81 +13,135 @@ class Nlopt < Formula
     sha256 "e3ea4064beff9f39ab2fed6a9e5f643a18fd7abb2b2302713ce7f58f970ceb09" => :sierra
   end
 
-  head do
-    url "https://github.com/stevengj/nlopt.git"
-    depends_on "cmake" => :build
-    depends_on "swig" => :build
-  end
-
-  depends_on "numpy"
+  depends_on "cmake" => [:build, :test]
 
   def install
-    ENV.deparallelize
+    args = *std_cmake_args + %w[
+      -DNLOPT_GUILE=OFF
+      -DNLOPT_MATLAB=OFF
+      -DNLOPT_OCTAVE=OFF
+      -DNLOPT_PYTHON=OFF
+      -DNLOPT_SWIG=OFF
```

Comment:
Are these options (`NLOPT_SWIG=OFF` and `NLOPT_PYTHON=OFF`) what allow us to drop the numpy and swig dependencies?

We want builds to be as featureful as possible; is there a good reason to build without python & swig?

Reply:
Upstream offer pip packages that offer a much better experience and support for more versions of Python.

Reply:
I can confirm that they have packages available for 2.6.1 and python 2.7 and 3.x. Thanks!